lib/rubocop/cop/bugcrowd/sleepy_specs.rb

```diff
@@ -9,29 +9,35 @@ module Bugcrowd
       #   expect(page).to have_content('blah')
       #
       #   # good
+      #   page.driver.wait_for_network_idle
+      #   expect(page).to have_content('blah')
+      #
+      #   # good
       #   expect(page).to have_content('blah', wait: 3)
       #
+      # use wait_for_network_idle when waiting for network requests
+      #
+      # https://github.com/rubycdp/cuprite#network-traffic
+      # https://github.com/rubycdp/ferrum#wait_for_idleoptions
+      #
+      # use wait when waiting for frontend only animations
       # https://github.com/teamcapybara/capybara#asynchronous-javascript-ajax-and-friends
       # https://www.varvet.com/blog/why-wait_until-was-removed-from-capybara/
       # https://www.reddit.com/r/rails/comments/25xrdy/is_there_a_way_to_change_capybaras_wait_time_just/
       #
+      # Other method of avoiding sleep:
       #
-      # Other methods of avoiding sleep:
-      # 1. Use an intermediate matcher
-      # ```
-      # expect(page).to have_content(flash_mesage) # the fast action
-      # expect(page).to have_content(activity_message) # the slow action
```

Comment:
Do you think this is now bad advice?

Reply:
The example looks like waiting for a network request so I would say not as reliable as `wait_for_network_idle`.

It might be relevant if you have a chain of frontend animations. I can try to update the example with something more meaningul

Reply:
Yeah in my head that example was like:

1. click button that opens modal
1. look for text in the modal to confirm the modal is open
1. look for a button within the modal
1. see the message saying it worked
1. click the modal and expect to see some backend change reflected on the front end eventually <--- this is the bit that can get pretty flaky and waiting for network idle is better

The only thing I'd say is you could probably have situations where you have live polling on an SPA and `wait_for_network_idle` just never ends up being true? I don't know if I've even encountered that but I'm guessing it could happen. Anyway 🤷🏼‍♂️ . Capybara should add some way of allowing you to add this to any call i.e.

```ruby
expect(page).to have_content(flash_mesage, wait_for_network_idle: true)
```

Reply:
😆 regardless this is y'alls repo now, my opinion is irrelevant. Miss you guys ❤️ Tell @zarakay I saw her Keybase message but was too lazy to follow up.

Reply:
I still want that feature :P. Miss you too